### ISSUE TITLE
Language specification for 'select' statement does not mention the 'break' statement at all!

### ISSUE THREAD

**Issue by ThrosturX** [golang/go#19456]
### What operating system and processor architecture are you using (`go env`)?
go version go1.7 linux/amd64

### What did you do?
I went to https://golang.org/ref/spec#Select_statements to see if `break` would break from select statements. Without reading the entire spec (or at least the grammar of `Statements` or the `Break statements` section), it is not easy to discern that the `break` statement truly breaks from a select. 

Although this isn't serious, this can cause misunderstandings like the following:

### What did you expect to see?
Running:
https://play.golang.org/p/YHTN69Jgnx
Expected output:
`Broke free`

### What did you see instead?
`process took too long`
(because the `break` statement didn't break from the for loop, as it shouldn't)

TL;DR: Things work fine, but the documentation could use some slight improvement, like maybe a footnote.

**Comment by ianlancetaylor** [comment]
At https://golang.org/ref/spec#Break_statements it says

    A "break" statement terminates execution of the innermost "for", "switch", or "select" statement
    within the same function.

There is no statement about `break` in the description of `for` or `switch`, so I don't think there needs to be one in the description of `select`.  That is the kind of thing that should be covered in the tour or a tutorial, not in the language spec.  Closing.
